<launch>
  <!-- 参数配置 -->
  <arg name="rgb_topic" default="/rgb/image_raw"/>
  <arg name="depth_topic" default="/depth_to_rgb/image_raw"/>
  <arg name="camera_info_topic" default="/rgb/camera_info"/>
  
  <!-- 启动3D检测节点 -->
  <node name="yolov8_3d_detector" pkg="yolo_graspnet_ros" type="detector_3d_node.py" output="screen">
    <param name="model_path" value="$(find yolo_graspnet_ros)/models/yolov8m.pt"/>
    <param name="rgb_topic" value="/rgb/image_raw"/>
    <param name="depth_topic" value="/depth_to_rgb/image_raw"/>
    <param name="camera_info" value="/rgb/camera_info"/>
  </node>
</launch>